src/iuse.cpp

```diff
@@ -4670,19 +4670,23 @@ cata::optional<int> iuse::dog_whistle( Character *p, item *it, bool, const tripo
     }
     p->add_msg_if_player( _( "You blow your dog whistle." ) );
 
-    std::array<std::string, 4> messages_friendly_or_neutral = {
-        _( "What is this unbearable sound!?" ),
-        _( "STOP.  MY EARS" ),
-        _( "I'm not a dog…" ),
-        _( "Would you kindly not do that?" )
+    std::array<const char *, 4> messages_friendly_or_neutral = {
```

Comment:
These should perhaps be arrays of `translation` objects so that the translations can be learnt evaluated. Then only one translation call will be needed when executing this code, rather than translating everything and only using one result.  Of course, I realise this is not your code, so maybe you don't want to spend time cleaning it up to that extent.